apps/website/README.md

```diff
@@ -0,0 +1,41 @@
+# Website
+
+This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
```

Comment:
Can we rename this directory to `apps/documentation` instead? `website` is a very generic name.

That way, we have

```
apps/documentation
apps/documentation/docs
```

Reply:
done!